src/main/java/com/spotify/reaper/storage/MemoryStorage.java

```diff
@@ -300,6 +304,31 @@ public boolean updateRepairSegment(RepairSegment newRepairSegment) {
   }
 
   @Override
+  public Collection<RepairParameters> getOngoingRepairsInCluster(String clusterName) {
```

Comment:
Is there anyway to make this method feel alittle bit less like being punched in the face with a boilerplate?  It just feels like all the logic is really obscured by  the surrounding verbosity.  A comment describing the algorithm at least?